src/Illuminate/Console/Command.php

```diff
@@ -414,7 +414,11 @@ public function info($string, $verbosity = null)
      */
     public function line($string, $style = null, $verbosity = null)
     {
-        $styled = $style ? "<$style>$string</$style>" : $string;
+        if ($style && str_contains($style, '=')) {
+            $styled = "<$style>$string</>";
+        } else {
+            $styled = $style ? "<$style>$string</$style>" : $string;
```

Comment:
@lucasmichot Yes, you are right, i dont want break the structure of current code. But i prefer you code.

The uniq problem is with ```Str::constains```, then be add:
```php
use Illuminate\Support\Str;
```

I you dont mind, i update the pr with you code.